### DISCUSSION TITLE
G-Helper ProcessList

### DISCUSSION THREAD

**Discussion by Existedz** [seerge/g-helper#3255] [Q&A]
Hello, I've been having problem with a file named ProcessList. This file randomly popped up on my desktop after using G-Helper. No matter how many time I delete it and move it to somewhere else, it keeps popping up on my Desktop. The information on the file contains nothing and blank. How do I solve this problem? I'm using TUF F16.
![Screenshot 2024-10-13 203952](https://github.com/user-attachments/assets/6d7477a0-4508-4b6a-9dd4-f6ee1ef18beb)

**Comment by seerge** [comment]
@Existedz hello, G-Helper is not creating that file :) it's some other app that you are using doing this.

Do you have any strange antiviruses like Malwarebytes or so that can leave that after checking G-Helper for example? I think they may do it.

**Reply by Existedz** [reply]
No sir, I don't have any antiviruses on my computer and I already uninstalled McAfee (Preinstalled). The reason that makes me tell you that G-Helper create that file is I deleted G-Helper and went back to Armoury Crate for a day and Process List File didn't appear on my desktop at that moment.

**Reply by seerge** [reply]
G-Helper doesn't create this file as I mentioned. Applications is completely transparent and open source (literally meaning you can check it's code and find what it does and what it does not).

So if that file appears - it means something else in your system creates that file. If it bothers you - you can just move GHelper.exe to any other folder rather than your desktop :)

**Reply by seerge** [reply]
@Existedz I quickly googled this, and some people mentioned it could be anticheat software in Valorant creating that file https://www.reddit.com/r/Asustuf/comments/1c78ux1/processlistini/

Do you play Valorant ? (or any other game that may use same anticheat)

**Reply by Existedz** [reply]
Yes sir, I do play Valorant. Do you have any idea on how to solve this problem without uninstalling Valorant?

**Reply by seerge** [reply]
@Existedz just move GHelper.exe to any other folder than your desktop

**Reply by Existedz** [reply]
Ok, I'll try. Thank you so much, seerge.

**Reply by Existedz** [reply]
The problem is now solved. I created G-Helper folder and moved it to the C Drive. Process List doesn't pop up again. Thank you so much.
